03week/towersOfHanoi.js

```diff
@@ -17,27 +29,65 @@ function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
-}
-
-function movePiece() {
-  // Your code here
-
-}
-
-function isLegal() {
-  // Your code here
+};
 
-}
+const movePiece = (startStack, endStack) => {
+  const start = stacks[startStack];
+  const end = stacks[endStack];
+  end.push(start.pop());
+};
 
-function checkForWin() {
-  // Your code here
+const isLegal = (startStack, endStack) => {
+  const SSLastNum = stacks[startStack][stacks[startStack].length-1];
```

Comment:
These variable names should start with a lowercase letter. The convention is upper case would signify a class.